math/src/main/java/com/powsybl/math/graph/UndirectedGraph.java

```diff
@@ -49,6 +56,10 @@
      */
     int getVertexCount();
 
+    default int getMaximumVertexIndex() {
```

Comment:
Add a Javadoc also there

Reply:
In the end I deleted the method since it basically `getVertexCapacity() - 1`